File: 프로그래머스/1/12910. 나누어 떨어지는 숫자 배열/나누어 떨어지는 숫자 배열.java
import java.util.ArrayList;
import java.util.Arrays;

class Solution {
    public int[] solution(int[] arr, int divisor) {
        int[] answer;
        
        ArrayList<Integer> list = new ArrayList<>();
        
        for (int value : arr) {
            if (value % divisor == 0) {
                list.add(value);
            }
        }
        
        if (list.size() == 0) {
            answer = new int[]{-1};
        } else {
            answer = new int[list.size()];
            for (int i = 0; i < list.size(); i++) {
                answer[i] = list.get(i);
            }
            Arrays.sort(answer);
        }
        
        return answer;
    }
}

File: 프로그래머스/1/12910. 나누어 떨어지는 숫자 배열/나누어 떨어지는 숫자 배열.js
function solution(arr, divisor) {
    let DividedByFiveList = arr.filter(num => num % divisor === 0).sort((a, b) => a - b)
    return DividedByFiveList.length >= 1 ? DividedByFiveList : [-1]
    // let answer = arr.filter(val => val % divisor === 0).sort((a, b) => a - b);
    // return answer.length > 0 ? answer : [-1];   
}